portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -142,5 +157,29 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
       return imagesService.getServingUrl(options);
     }
   }
-}
+  
+  /** 
+   * Returns the score of the sentiment of the given message, 
+   * which is a float from -1 to 1 representing how negative or positive the text it.
+   */
+  private float getSentimentScore(String message) throws IOException {
+    Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    languageService.close();
+
+    return score;
```

Comment:
I would reorder this and combine lines.
Unless you need the `languageService` open for the `getScore()`?
```suggestion
    languageService.close();
    return sentiment.getScore();
```

Reply:
that worked